docs/pipelines/library/service-endpoints.md

```diff
@@ -66,53 +66,94 @@ Service connections are created at project scope. A service connection created i
 
 1. Choose **OK** to create the connection.
 
-> For more information about Azure Resource Manager service connections, see [Connect to Microsoft Azure](connect-to-azure.md).
-> You can also create your own [custom service connections](../../extend/develop/service-endpoints.md).
-
 ## Manage a service connection
 
 1. In Azure DevOps, open the **Service connections** page from the [project settings page](../../project/navigation/go-to-service-page.md#open-project-settings).
    Or, in TFS, open the **Services** page from the "settings" icon in the top menu bar.
 
 1. Select the service connection you want to manage.
 
-1. Choose from the list of **Actions** in the **Details** tab in the right pane.
-
-The actions available depend on the chosen type of connection. You can update only
-some properties of connections; for example, to change the selected subscription
-you must re-create the connection. Choose **Disconnect** to delete or remove a connection.
+1. You will land in the **Overview** tab of the service connection where you can see the details of the service connection i.e. type, creator, authentication schema etc.
```

Comment:
authentication schema  - user may not relate to this

Reply:
Updated as suggested